src/frontend/src/tests/mocks/tokens.mock.ts

```diff
@@ -0,0 +1,9 @@
+import type { Token } from '@dfinity/utils';
+
+export const ICP_SYMBOL = 'ICP';
+
+export const ICP_TOKEN: Token = {
```

Comment:
why not using existing env constant `ICP_TOKEN`?

Reply:
I tried at first, but it raises an issue with some other env variables that need to be fixed. For example VITE_APP_VERSION. So, to simplify the tests, it's better to mock, since we only need it as constant value to mock input and returns.

Reply:
If one day we got time, I think it would be worth a try not to mock it. It can be particularly useful if we start reusing some test for unit and E2E tests.